navigation_ptp_time: 18
navigation_ptp_timeout: 60
